recipes/pulseaudio/all/conanfile.py

```diff
@@ -62,15 +62,15 @@ def requirements(self):
         self.requires("libsndfile/1.0.30")
         self.requires("libcap/2.45")
         if self.options.with_alsa:
-            self.requires("libalsa/1.1.9")
+            self.requires("libalsa/1.2.4")
         if self.options.with_glib:
             self.requires("glib/2.64.0")
         if self.options.with_fftw:
             self.requires("fftw/3.3.8")
         if self.options.with_x11:
             self.requires("xorg/system")
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1h")   
+            self.requires("openssl/1.1.1i")
         # FIXME: enable when #2147 is merged
         # if self.options.with_dbus
         #     self.requires("dbus/1.12.16")
```

Comment:
https://github.com/conan-io/conan-center-index/pull/3950

```suggestion
        # FIXME: enable when #3950 is merged
        # if self.options.with_dbus
        #     self.requires("dbus/1.12.16")
```

Reply:
There's two other occurrences outside of the diff

Reply:
You are right, but I'm not going to invalid reviews for that change